swap the contributor role type and the name fields in deposit form

peetucket opened the pull request:
## Why was this change made?

Fixes #297


## How was this change tested?

Localhost browser


## Which documentation and/or configurations were updated?

jcoyne commented on app/components/works/contributor_row_component.html.erb:
```diff
@@ -6,6 +6,18 @@
+                    } %>
+  </div>
+  <div class="col-md-1 role-prefix">
+    is
```
Does this still make sense?   @astridu?  @peetucket maybe add a screenshot.

peetucket commented:
Added screenshot.  Changed the word "as" to "is" ... seemed to make more sense to me.

![Screen Shot 2020-11-05 at 12 10 30 PM](https://user-images.githubusercontent.com/47137/98291448-fe13d980-1f5f-11eb-8382-b2f4827cfe6b.png)

astridu commented:
This looks good, and I agree, "is" is more appropriate. Thanks!
